### PR TITLE
Added new Prometheus Rule Groups type

### PR DESCRIPTION
**Pull request by giladsu**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Add a changelog entry for this PR by answering the following questions:
  1. What's the purpose of the update?
      - [x] new service onboarding
      - [ ] new API version
      - [ ] update existing version for new feature
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify
  2. When are you targeting to deploy the new service/feature to public regions? Please provide the date or, if the date is not yet available, the month.
  3. When do you expect to publish the swagger? Please provide date or, the the date is not yet available, the month.
  4. If updating an existing version, please select the specific langauge SDKs and CLIs that must be refreshed after the swagger is published.
      - [ ] SDK of .NET (need service team to ensure code readiness)
      - [ ] SDK of Python
      - [ ] SDK of Java
      - [ ] SDK of Js
      - [ ] SDK of Go
      - [ ] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [ ] No refresh required for updates in this PR

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist 

> **Applicability**: :warning: 
>
> If your changes encompass only the following scenarios, you should SKIP this section, as these scenarios do not require ARM review.
> - Change to data plane APIs 
> - Adding new properties 
> - All removals

Otherwise your PR may be subject to ARM review requirements. Complete the following:
- [x] Check this box if any of the following apply to the PR so that label “WaitForARMFeedback” will be added automatically to begin ARM API Review. Failure to comply may result in delays to the manifest.
  - Adding a new service
  - Adding new API(s)
  - Adding a new API version
    -[ ] To review changes efficiently, ensure you copy the existing version into the new directory structure for first commit and then push new changes, including version updates, in separate commits.

- [x] Ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If any of the following scenarios apply to the PR, request approval from the Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 
- [ ] Removing API(s) in a stable version
- [ ] Removing properties in a stable version
- [ ] Removing API version(s) in a stable version
- [ ] Updating API in a stable or public preview version with Breaking Change Validation errors
- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by msyyc** [comment]
Hi @giladsu Please follow the advice of `openapi-workflow-bot` to pass `Required` CI
image

**Comment by giladsu** [comment]
@msyyc - I've fixed all validation issues except for one that I am unable to reproduce with the validation tools.  IIUC, the error says that the examples contain the "location" property which is not defined in the swagger, but I can see that it is.

Also, running oav validate-example locally does not produce this error.

Do you know what the issue is or how I can investigate it further?

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json
Comment (mentat9) [inline specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json:483]: >record [](http://example.com/codeflow?start=9&length=6)

Consider a more descriptive name here, maybe recordingRuleName, ruleName, displayName, or similar.

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json
Comment (mentat9) [inline specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json:487]: >alert [](http://example.com/codeflow?start=9&length=5)

alertName?

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json
Comment (mentat9) [inline specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json:495]: >severity [](http://example.com/codeflow?start=9&length=8)

Consider an enum that identifies all valid severities, e.g. [ Informational, Warning, Error, Critical ], or whatever values you need.

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json
Comment (mentat9) [inline specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json:498]: >the expression to run for the rule [](http://example.com/codeflow?start=26&length=34)

Fix description.

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json
Comment (mentat9) [inline specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json:500]: >for [](http://example.com/codeflow?start=9&length=3)

fireAfterSeconds (or similar)?

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json
Comment (mentat9) [inline specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json:506]: >additionalProperties [](http://example.com/codeflow?start=11&length=20)

ARM recommendation is to avoid this pattern wherever possible, since it doesn't allow reasoning about the payload (so things like Azure Policy and ARG can't use them). Same for annotations. If these properties are always only user-provided, then it won't block you, but it's still worth considering other options wherever possible.

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json
Comment (mentat9) [inline specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json:545]: >PrometheusRuleGroupsErrorResponse [](http://example.com/codeflow?start=5&length=33)

Recommend using the ErrorResponse type in common-types:

"$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse".

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json
Comment (mentat9) [inline specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json:569]: >parameters [](http://example.com/codeflow?start=3&length=10)

The first 3 of these can be pulled from common-types.

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json
Comment (mentat9) [inline specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json:422]: >Resource [](http://example.com/codeflow?start=33&length=8)

You can use TrackedResource from common-types for this.

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json
Comment (mentat9) [inline specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json:409]: >lastUpdatedTime [](http://example.com/codeflow?start=9&length=15)

You should be able to remove this if it's the same as the systemData.LastModifiedAt property.

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json
Comment (mentat9) [inline specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json:398]: >interval [](http://example.com/codeflow?start=9&length=8)

What kind of values does this property take? I think this can be made much easier to use. If there are a set of known intervals, it works better as an enum, so they can be easily discovered. If the set isn't fixed, the the name a description need to explain what these values can be (e.g. what are the units, how is an interval measured, etc.).

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json
Comment (mentat9) [inline specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json:372]: >additionalProperties [](http://example.com/codeflow?start=11&length=20)

See other comments re: additionalProperties pattern.

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json
Comment (mentat9) [inline specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json:366]: >actionGroupId [](http://example.com/codeflow?start=9&length=13)

What is the scope on this id?

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json
Comment (mentat9) [inline specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json:30]: >prometheusRuleGroups [](http://example.com/codeflow?start=74&length=20)

Prometheus sounds like a project name. Can you use a more descriptive name for this resource? I don't think the name Prometheus should appear anywhere in this API definition, since (AFAICT) nobody outside of your RP would know what it refers to.

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json
Comment (mentat9) [inline specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json:538]: >timeToResolve [](http://example.com/codeflow?start=9&length=13)

secondsUntilResolution? In any case, consider including the units in the name and description.

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json
Comment (mentat9) [inline specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json:534]: >autoResolved [](http://example.com/codeflow?start=9&length=12)

ARM recommends string enums over boolean. Generally enums make better properties than booleans. They are more descriptive, flexible and future-proof while being easier for customers to discover, understand and use.

In this case, consider enum resolutionType: [ Manual, Automatic ] (or similar).

**Comment by mentat9** [comment]
Are you onboarding these APIs using RPaaS? If not, why not?

**Comment by msyyc** [comment]
> @msyyc Yuchao Yan FTE - I've fixed all validation issues except for one that I am unable to reproduce with the validation tools. IIUC, the error says that the examples contain the "location" property which is not defined in the swagger, but I can see that it is.
> 
> Also, running oav validate-example locally does not produce this error.
> 
> Do you know what the issue is or how I can investigate it further?

Hi @giladsu If any question about CI, please contact swagger teams:
image

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json
Comment (giladsu) [reply specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json:483]: @mentat9 - we totally agree, but this resource type is for an Azure offering of an existing OSS monitoring system and we have decided to align as much as possible with the existing system to decrease learning curve of customers.

More information here: https://prometheus.io/docs/prometheus/latest/configuration/recording_rules/

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json
Comment (giladsu) [reply specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json:487]: Alert is terminology for Prometheus system: https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json
Comment (giladsu) [reply specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json:498]: Done.

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json
Comment (giladsu) [reply specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json:500]: "For" is from Prometheus system: https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json
Comment (giladsu) [reply specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json:506]: Agreed - but again this is due to alignment with Prometheus: https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json
Comment (giladsu) [reply specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json:545]: Updated.

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json
Comment (giladsu) [reply specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json:422]: Thanks.

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json
Comment (giladsu) [reply specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json:372]: This is for alignment with metric alerts

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json
Comment (giladsu) [reply specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json:366]: Not sure I understand the question - it is the actionGroupId of the customer's ARM action group that the system should use for sending notifications

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json
Comment (giladsu) [reply specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json:30]: It is alignment with the OSS naming, which is well known in to potential customers.

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json
Comment (giladsu) [reply specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json:495]: Thanks mentat - we like this suggest a lot but need to keep this as int for alignment with other alerting resource types (metric alerts, log search alerts, etc.)

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json
Comment (giladsu) [reply specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json:569]: Done. Thanks.

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json
Comment (giladsu) [reply specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json:409]: Thanks.

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json
Comment (giladsu) [reply specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json:398]: This is a string represented in ISO 8601 duration format (alignment with other alert rule types). I've update the field to include the string format expected.

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json
Comment (giladsu) [reply specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json:538]: Same response as "interval"

**Comment by giladsu** [comment]
> Are you onboarding these APIs using RPaaS? If not, why not?

Currently no - this is being onboarded to our generic alert rule control plane service.  We have discussed moving this to RPaaS in the future (and PrometheusRuleGroups along with it of course), but do not have a concrete timeline.

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json
Comment (giladsu) [reply specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json:495]: We need to keep this as an int for alignment with other alert rule types

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json
Comment (giladsu) [reply specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json:534]: This is also an alignment with metric alerts and log search alerts

**Comment by giladsu** [comment]
@mentat9 - thank you for your comments.  I've addressed all of them except for one where I need some more details in order to answer.

Please let me know if I can provide any more information.

Thanks.

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json
Comment (mentat9) [reply specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json:545]: You should remove this and the next definition since they aren't used any more.

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json
Comment (mentat9) [reply specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json:409]: I don't see an update. Are you planning to remove it?

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json
Comment (mentat9) [reply specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json:398]: Did't see an update.

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json
Comment (mentat9) [reply specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json:366]: Sorry, I meant is it an ARM resource ID, or an ID assigned by some other system... something else?

**Comment by mentat9** [comment]
> @mentat9 - thank you for your comments. I've addressed all of them except for one where I need some more details in order to answer.
> 
> Please let me know if I can provide any more information.
> 
> Thanks.

OK, thanks @giladsu. Most comments are resolved: a small handful still active.

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json
Comment (giladsu) [reply specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json:366]: Ah - yes it is an ARM resource ID of an actiongroup

**Comment by giladsu** [comment]
> > @mentat9 - thank you for your comments. I've addressed all of them except for one where I need some more details in order to answer.
> > Please let me know if I can provide any more information.
> > Thanks.
> 
> OK, thanks @giladsu. Most comments are resolved: a small handful still active.

@mentat9 - thanks. I've answered the last one.  If there is something else I am missing or haven't answered, please let me know.

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json
Comment (mentat9) [reply specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json:366]: Consider naming this actionGroupResourceId for clarity.

**Comment by mentat9** [comment]
> > > @mentat9 - thank you for your comments. I've addressed all of them except for one where I need some more details in order to answer.
> > > Please let me know if I can provide any more information.
> > > Thanks.
> > 
> > 
> > OK, thanks @giladsu. Most comments are resolved: a small handful still active.
> 
> @mentat9 - thanks. I've answered the last one. If there is something else I am missing or haven't answered, please let me know.

Your comments indicate you have made updates, but you haven't pushed any changes. Also, there are still 4 unresolved comments.

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json
Comment (giladsu) [reply specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json:366]: ActionGroupId is the property name for metric alerts and log search alerts, so we need to keep for alignment

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json
Comment (giladsu) [reply specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json:545]: Thanks - done.

**Comment by giladsu** [comment]
Currently no - this is being onboarded to our generic alert rule control plane service. We have discussed moving this to RPaaS in the future (and PrometheusRuleGroups along with it of course), but do not have a concrete timeline.

---
In reply to: [958603876](https://github.com/Azure/azure-rest-api-specs/pull/16613#issuecomment-958603876) [](http://example.com/codeflow?ancestors=958603876,958603876)

**Comment by giladsu** [comment]
@mentat9 - apologies, didn't push the final changes in.  everything should be updated now, but please let me know if I missed anything.

---
In reply to: [964454703](https://github.com/Azure/azure-rest-api-specs/pull/16613#issuecomment-964454703) [](http://example.com/codeflow?ancestors=964454703,964454703)

**Comment by giladsu** [comment]
@msyyc - can this be signed off by you to continue with the merge or are there other steps that need to be taken?

**Comment by msyyc** [comment]
Hi @giladsu, you need to ask for at least one approval from assigners before merge. Usually, assignee can't sign off the content.
image

**Comment by giladsu** [comment]
@mentat9 - I'm not sure why but the 2 "reviewers" listed are not part of the team and it doesnt seem possible to add additional reviewers.  Can you sign off or advise on how to add another reviewer to the list?

**Comment by mentat9** [comment]
> @mentat9 - I'm not sure why but the 2 "reviewers" listed are not part of the team and it doesnt seem possible to add additional reviewers. Can you sign off or advise on how to add another reviewer to the list?

@giladsu, ARM reviewers aren't authorized to approve or merge PRs. We just review and approve changes to the REST layer. If your PR isn't getting traction, you can try email to SDK Release Owners (sdkowners@microsoft.com) to get some help.

**Review by msyyc — APPROVED** [review]

**Pull request merged**
